clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -113,6 +113,14 @@ service LuceneServer {
       body: "*"
     };
 
+    }
+    /* Sync an index state */
+    rpc reloadIndexState (ReloadIndexStateRequest) returns (DummyResponse) {
```

Comment:
Since this is no longer reloading just the index state, perhase the endpoint name could be changed. This would also be true for some of the other methods/resources.

Also, it might be good to create a placeholder response message type instead of using `DummyResponse`. I could imagine wanting to return something like the current state generations in the future.